docs/guide/personalization.md

```diff
@@ -562,26 +552,10 @@ For example, to retrieve the `rss` variation of the image, use:
 
 ## Troubleshooting
 
-### "Console error: eZ is not defined"
-
-If you see this error in the console of your browser's debugger you need to rerun the asset installer:
-
-``` bash
-php app/console assets:install --symlink --env=prod
-```
-
-### Authentication does not work
-
-If authentication fails although you added a user in your local directory please check if you used double quotes around the username in the export settings.
-
-### Requested content type is not supported (CONFLICT)
-
-When fetching recommendations you get a message that the requested content type is not supported (CONFLICT).
-Take a look at the scenario configuration in the [Admin Dashboard](https://admin.yoochoose.net) and check if the input and output types are properly set.
-
 ### Logging
 
-Most operations are logged via the `ez_recommendation` [Monolog channel](http://symfony.com/doc/current/cookbook/logging/channels_handlers.html). To log everything about Recommendation to `dev.recommendation.log`, add the following to your `config.yml`:
+Most operations are logged via the `ez_recommendation` [Monolog channel](http://symfony.com/doc/current/cookbook/logging/channels_handlers.html).
```

Comment:
current -> 4.x (to be specified)

Reply:
It's not 100% certain which Symfony version 3.0 will be based on, so I think we can leave current for now and then fix all links together.